underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -264,14 +270,13 @@ public long getSpace(String path, SpaceType type) throws IOException {
 
   @Override
   public boolean isFile(String path) throws IOException {
-    String strippedPath = stripPrefixIfPresent(path);
-    return mAccount.getContainer(mContainerName).getObject(strippedPath).exists();
+    return exists(path) && !isDirectory(path);
```

Comment:
`isFile` calls `exists` then `isDirectory`. They both will call swift apis, which means it now does multiple rpcs. The previous code looks like it does only 1? Is this expected?